config/exporters/monitoring/grafana/base/prometheus-exporter-service-monitor.yaml

```diff
@@ -24,7 +24,7 @@ rules:
   - get
 ---
 apiVersion: rbac.authorization.k8s.io/v1
-kind: RoleBinding
+kind: ClusterRoleBinding
 metadata:
   name: exporter-role-binding-metrics-reader
   namespace: appstudio-grafana-datasource-exporter
```

Comment:
I think the `namespace` will be dropped once this is deployed and it might keep ArgoCD constantly out of sync.
```suggestion

```